### PR TITLE
require address for members signing up

### LINKED ISSUE #1312: Require Address for Members Signing Up
For tax reasons we require new members' addresses. This needs to be incorporated into the new member signup flow before we run their subscription through Stripe.

### PR DESCRIPTION
**Pull request by ShadyNagy**
Closes #1312

### CODE REVIEW

File: src/DevBetterWeb.Core/Entities/Member.cs [hunk 1]
```diff
@@ -123,11 +126,14 @@ public void UpdateName(string? firstName, string? lastName, bool isEvent = true)
...
 
-	public void UpdateAddress(string? address, bool isEvent = true)
+	public void UpdateAddress(string? address, string? city, string? country, string? postalCode, bool isEvent = true)
 	{
 		if (Address == address) return;
 
 		Address = address;
+		City = city;
+		Country = country;
+		PostalCode = postalCode;
```
Comment (lloydkevin) [inline src/DevBetterWeb.Core/Entities/Member.cs:137]: Do we need similar equality checks for the new fields like in `address`?

Comment (ShadyNagy) [reply src/DevBetterWeb.Core/Entities/Member.cs:156]: This checks will not work in the same way need to be inverted.

**Comment by ShadyNagy** [comment]
The Shipping Address is used.

**Comment by ShadyNagy** [comment]
@ardalis 
It's ready for merge.

**Comment by ardalis** [comment]
🤞

**Pull request merged**